coverage.rst

```diff
@@ -72,30 +72,53 @@ Using coverage.py
 One of the most popular third-party coverage tools is `coverage.py`_ which
 provides very nice HTML output along with advanced features such as
 :ref:`branch coverage <branch_coverage>`. If you prefer to stay with tools only
-provided by the stdlib then you can by :ref:`using test.regrtest
+provided by the stdlib then you can :ref:`use test.regrtest
 <coverage_by_regrtest>`.
 
-Because the in-development version of Python is bleeding-edge, it is possible
-that the latest release version of coverage.py will not work. In that case you
-should try using the in-development of coverage.py to see if it has been
-updated as needed. To do this you should clone/check out the development version
-of coverage.py::
+
+.. _install_coverage:
+
+Install Coverage
+''''''''''''''''
+
+By default, pip will not install into the in-development version of Python you 
+just built, and this built version of Python will not see packages installed 
+into your default version of Python. One option is to use a virtual environment
+to install coverage::
+
+    ./python.exe -m venv ../cpython-venv
+    source ../cpython-venv/bin/activate
+    pip install coverage
+
+You can now use python without the ./ for the rest of these instructions, as long as
+your venv is activated. For more info on venv see https://docs.python.org/3/tutorial/venv.html.
```

Comment:
Instead of `https://docs.python.org/3/tutorial/venv.html`
use
```
`Virtual Environment <https://docs.python.org/3/tutorial/venv.html>`_ documentation.
```